central/compliance/datastore/sac_filter.go

```diff
@@ -107,7 +107,7 @@ func (ds *sacFilterImpl) filterClusters(ctx context.Context, clusters set.String
 	// Filter the compliance results by cluster.
 	allowed := set.NewStringSet()
 	for cluster := range clusters {
-		if resourceScopeChecker.TryAllowed(sac.ClusterScopeKey(cluster)) {
+		if ok, _ := resourceScopeChecker.Allowed(ctx, sac.ClusterScopeKey(cluster)); ok {
```

Comment:
Would it make sense to propagate the potential error from the Allowed function ? Or does `Allowed` always return `nil` anyway and is being re-factored in a later pull request ?

Reply:
It always returns `nil` and will be refactored in https://github.com/stackrox/stackrox/pull/2780